docs/GTest-friend.md

```diff
@@ -0,0 +1,57 @@
+There already are many good resources which explains why you shouldn't test private parts of your implementation and how to avoid it.
+
+If you want to make make your classes friend with their test, here are some ways to achieve it:
+
+* <font size="1">Add `#define private public` & `#define protected public` before including production headers to your test sources. But I never said this.</font>
+
+* You can declare your test with an existing class instead of a string :
+`GTEST(SomeClass) { ... }`
+With this syntax, the test will inherit from `SomeClass`, so you can access to protected members.
+
+    This is the least intrusive way, as there is nothing added to the tested class - apart from maybe replace private by protected.
+
+* You can declare your class `friend` with the test class `GTEST(SomeClass)` :
+    ```// Forward declare some GUnit classes
+    template <typename ...> struct GTEST;
+    namespace testing::detail { template <char ...> struct string; }
+    
+    class SomeClass {
+        friend struct GTEST<SomeClass, testing::detail::string<> >;
+    };
+
+    GTEST(Game) { ... }
+    ```
+* You can also declare friend with the test class `GTEST("SomeTest")`, it is more cumbersome :
```

Comment:
Perhaps favor `although it is more cumbersome` or `though it is more cumbersome`?